src/test/java/com/conveyal/datatools/manager/jobs/GisExportJobTest.java

```diff
@@ -87,7 +91,10 @@ public void canExportStops () throws IOException {
         GisExportJob gisExportJob = new GisExportJob(GisExportJob.ExportType.STOPS, zipFile, ids, "test");
         gisExportJob.run();
         assertThat(gisExportJob.status.error, equalTo(false));
-        FeatureCollection collection = getFeatureCollectionFromZippedShapefile(zipFile);
+        File[] files = getFilesFromZippedShapefile(zipFile);
+        FeatureCollection collection = getFeatureCollectionFromZippedShapefile(files);
+        CoordinateReferenceSystem crs = getCRSFromShapefiles(files);
+        assertThat("Coordinate reference system is not null.", crs, notNullValue());
```

Comment:
Seems like these 4 lines are re-used a bunch and could be made into a method.

Reply:
I didn't want to bury the assertion in a function that grabbed the feature collection, but I have gone ahead and refactored the CRS check into a shared method.